### PR TITLE
Fix ESLint errors in EmbeddedYoutubeVideo, SideNav, TestingNotice, and QuickstartWrapper

### PR DESCRIPTION
**Pull request by mjlumetta**
### Description

fixes ESLint errors in the following components for [API-1323](https://vajira.max.gov/browse/API-1323).

- `SideNav` and `SideNavEntry`
- `TestingNotice`
- `EmbeddedYoutubeVideo`
- `QuickstartWrapper`

also moves `EmbeddedYoutubeVideo` to its own subdirectory.

### Process

- [n/a] Tests added or updated for change
- [n/a] Docs updated
- [n/a] Accessibility concerns have been considered and addressed

### CODE REVIEW

**Review by ediiotero — DISMISSED** [review]
looks good

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "build": "node scripts/build.js",
     "lint": "./node_modules/.bin/tslint --config tslint.json --project tsconfig.json",
     "lint:ci": "./node_modules/.bin/tslint --config tslint.ci.json --project tsconfig.json -t junit > lint-results.xml",
-    "eslint": "./node_modules/.bin/eslint --ext .ts,.tsx src/actions src/apiDefs src/components/onlyTags src/containers/documentation/swaggerPlugins src/containers/releaseNotes src/containers/support src/reducers",
+    "eslint": "./node_modules/.bin/eslint --ext .ts,.tsx src/actions src/apiDefs src/components/onlyTags src/containers/documentation/swaggerPlugins src/containers/releaseNotes src/containers/support src/reducers src/types",
```
Comment (nathanbwright) [inline package.json:11]: Should `src/components/sideNav` be added here too? Based on your changes to `src/components/sideNav/SideNavEntry.tsx`

Comment (mjlumetta) [reply package.json:11]: i've given up on adding individual directories within `src/components` to the ESLint command at this point. whichever one comes last between this PR and #501, i'll add the whole `src/components` directory when i merge the upstream changes back into the PR branch.

**Review by nathanbwright — APPROVED** [review]
👍

**Pull request merged**